### PR TITLE
Swift 3.0

### PR DESCRIPTION
**Pull request by johncederholm**
Updated for Swift 3.0

### CODE REVIEW

File: Sources/RepetitiveTask/RepetitiveTaskError.swift [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-public enum RepetitiveTaskError: ErrorType {
-    case Failed(NSError)
+public enum RepetitiveTaskError: Error {
+    case Failed(Error)
     case RetryFailed(Int)
     case RetryDelay(Double)
     case NoData
-
+
```
Comment (krzyzanowskim) [inline Sources/RepetitiveTask/RepetitiveTaskError.swift]: trim spaces please.

**Review by krzyzanowskim — APPROVED** [review]
looks good to me. thanks! small formatting nitpick if you can update before merge. thanks!

**Comment by johncederholm** [comment]
First, thank you for putting up with a pull request for a very old repo. I am trying to get caught up with github and this repo seemed like a good one with which to start.
Second, do you not end files with newlines? Please let me know if my new branch did that correctly. Thanks.

**Comment by krzyzanowskim** [comment]
@johncederholm I've meant "    " spaces.

**Comment by krzyzanowskim** [comment]
Good job, thanks!

**Pull request merged**